### DISCUSSION TITLE
CUTLASS in PaddlePaddle

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#688] [General]
CUTLASS [example 36](https://github.com/NVIDIA/cutlass/tree/master/examples/36_gather_scatter_fusion) gather + gemm + scatter fusion kernel was integrated into [paddlepaddle](https://github.com/PaddlePaddle/Paddle) by @umiswing as a new way to do sparse gemm.

**Comment by umiswing** [comment]
Yes, I use this fusion to accelerate sparse convolution in paddlepaddle. :)
